### ISSUE TITLE
Test the program

### ISSUE THREAD

**Issue by aThorp96** [aThorp96/savingsApp#6]
Look into code tests for the backend.

**Comment by aThorp96** [comment]
@graysonfenwick I'm not sure if you have any experience writing tests, but I think it'd be helpful if one of us gave if a whack at some point. I think it's a little superfluous for a project like this, but it could be good experience.

**Comment by graysonfenwick** [comment]
Yeah I definitely agree that tests are a good idea, they don't have to be particularly complex but still good to have

**Comment by graysonfenwick** [comment]
And it looks like XCode on Mac auto-generates unit tests, so that'll be helpful
